client/src/Movies/Movie.js

```diff
@@ -61,8 +60,12 @@ export default class Movie extends Component {
             </div>
           ))}
         </div>
-        <div className="save-button">Save</div>
+        <div onClick={this.saveMovie} className="save-button">Save</div>
       </div>
     );
   }
 }
+
+
+
+// window.location.href.split('/').slice(-1)
```

Comment:
I still find this so cool bc you're the only person who went looking and found a unique way to do it. Your google-fu is strong! This mentality will make you a great dev!